app/models/solidus/affirm/checkout.rb

```diff
@@ -0,0 +1,4 @@
+module Solidus
+  class Affirm::Checkout < ActiveRecord::Base
```

Comment:
Ah, that might explain the load issues that I did see now and then. Wasn't aware of that!